core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/accountlist/AccountListViewModel.kt

```diff
@@ -143,6 +149,18 @@ class AccountListViewModel(
         }
     }
 
+    fun onClickAppShare(shareLink: Boolean) {
+        viewModelScope.launch {
+            try {
+                shareAppUseCase?.invoke(shareLink)
+            } catch (e: IllegalArgumentException) {
```

Comment:
The catch clauses are exactly the same: you only need one catch e. Best practice now is to use Throwable instead of Exception.